src/ui/profile_preferences.ui

```diff
@@ -3729,6 +3736,7 @@ you can use it but there could be issues with aligning columns of text</string>
   <tabstop>ircNick</tabstop>
   <tabstop>ircHostPort</tabstop>
   <tabstop>ircChannels</tabstop>
+  <tabstop>ircHostSecure</tabstop>
```

Comment:
As the tab order moves down the left column before jumping back to the start of the right column of controls in this block you need to reorder things so the tab order remains in that manner with the extra new control:
```suggestion
  <tabstop>ircHostSecure</tabstop>
  <tabstop>ircHostPort</tabstop>
  <tabstop>ircChannels</tabstop>
```